Core/Games/KerbalSpaceProgram/GameVersionProviders/KspBuildMap.cs

```diff
@@ -106,20 +112,28 @@ private bool TrySetBuildMap(string buildMapJson)
                 return false;
             }
         }
+
+        private bool TrySetCachedBuildMap()
+        {
+            try
+            {
+                Log.Debug("Getting cached build map");
+                return TrySetBuildMap(File.ReadAllText(cachedBuildMapPath));
+            }
+            catch (Exception e)
```

Comment:
Are we doing anything with this exception we're catching? We're getting a few notes about it.

Reply:
No; unlike the other `TrySet*BuildMap` functions, we don't want to print a warning for this one because we _expect_ it to be missing the first time. I've removed this variable.